### Notebook Diffs vs origin/main
nbdiff /dev/null newnew.ipynb
--- /dev/null  2025-07-30 14:11:31.246634
+++ newnew.ipynb  2025-07-30 14:11:09.449273
## added /cells:
+  code cell:
+    source:
+      new new 




